app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const {getAllTopics} = require("./controllers/topics.controllers")
+const {invalidPath} = require("./error-handling")
+
+const app = express();
+
+app.use(express.json());
```

Comment:
we don't need this line yet, can you remove it until the code requires you to put it in